### PR TITLE
Split tables using certain test results.

### PR DESCRIPTION
**Pull request by Mrcarroll66**
Adds functionality to split the filtered list to 2 lists of x length

List 1 and List 2 are build using a single new component; SplitListTable

https://github.com/stdlib-js/stats-ttest2 is used to perform the ttest on the selected columns

### CODE REVIEW

File: src/components/page_components/splitListTable.js
Comment (jwranahan) [inline src/components/page_components/splitListTable.js:1]: Console logs left at lines 82 and 87 could be taken out, everything works properly otherwise.

Comment (jwranahan) [inline src/components/page_components/splitListTable.js:1]: Console logs left at lines 82 and 87 could be taken out, everything works properly otherwise.

**Review by jwranahan — APPROVED** [review]
Everything works well and is readable.

**Review by jwranahan — APPROVED** [review]
Everything works well and is readable.

File: src/components/page_components/splitListTable.js
Comment (Mrcarroll66) [reply src/components/page_components/splitListTable.js:1]: I think it's okay to leave them for debugging purposes

**Pull request merged**